ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py

```diff
@@ -834,3 +834,44 @@ def test_jax_numpy_cond(
         x=x[0],
         p=p,
     )
+
+
+# lstsq
+@handle_frontend_test(
+    fn_tree="jax.numpy.linalg.lstsq",
+    dtype_and_data=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0,
+        max_value=10,
+        shape=helpers.ints(min_value=2, max_value=5).map(lambda x: tuple([x, x + 1])),
+    ).filter(
+        lambda x: "float16" not in x[0]
+        and "bfloat16" not in x[0]
```

Comment:
You shouldn't exclude dtypes in `filter()`, it should be the job of `get_dtypes`. Please remove it and if you still have a problem let me know or ask in the `ivy-tests` channel in our [Discord](https://discord.com/invite/G4aR9Q7DTN)